package.json

```diff
@@ -73,7 +76,10 @@
     "xhr-mock": "^2.5.1"
   },
   "dependencies": {
-    "@looker/chatty": "^2.3.2"
+    "@looker/chatty": "^2.3.2",
+    "@looker/sdk": "^22.16.0",
+    "@looker/sdk-node": "^22.16.0",
```

Comment:
Does `sdk-node` need to be a dependency or can it be a devDependency? Wondering about node pollution in the browser

Reply:
Good catch! Updated